singledop/__init__.py

```diff
@@ -0,0 +1,2 @@
+from singledop import SingleDoppler2D
+from singledop import AnalysisDisplay
```

Comment:
Should add as well:
from singledop import SaveFile
from singledop import NetcdfSave